src/web/impl/WsBase.hpp

```diff
@@ -102,22 +101,19 @@ class WsBase : public ConnectionBase, public std::enable_shared_from_this<WsBase
 
 public:
     explicit WsBase(
+        util::Config const& config,
         std::string ip,
         std::reference_wrapper<util::TagDecoratorFactory const> tagFactory,
         std::reference_wrapper<dosguard::DOSGuardInterface> dosGuard,
         std::shared_ptr<HandlerType> const& handler,
         boost::beast::flat_buffer&& buffer
     )
-        : ConnectionBase(tagFactory, ip)
-        , messagesLength_(PrometheusService::gaugeInt(
-              "ws_messages_length",
-              util::prometheus::Labels(),
-              "The total length of messages in the queue"
-          ))
-        , buffer_(std::move(buffer))
-        , dosGuard_(dosGuard)
-        , handler_(handler)
+        : ConnectionBase(tagFactory, ip), buffer_(std::move(buffer)), dosGuard_(dosGuard), handler_(handler)
     {
+        // If the transactions number is 200 per ledger, A client which subscribes everything will send 400 feeds for
+        // each ledger. Assume 2s per ledger, we allow user delay 80000/400 * 2 = 400s
+        maxSendingQueueSize_ = config.valueOr("ws_max_sending_queue_size", 80000);
```

Comment:
This should be initialised before `{`, otherwise there is an uninitialised field for a bit.